bundles/org.openhab.persistence.dynamodb/src/main/java/org/openhab/persistence/dynamodb/internal/DynamoDBHistoricItem.java

```diff
@@ -48,7 +49,12 @@ public String getName() {
 
     @Override
     public ZonedDateTime getTimestamp() {
-        return timestamp;
+        return instant.atZone(ZoneId.systemDefault());
```

Comment:
Not sure if i feel fully confident with this. This timezone is the system timezone, this may be different to the openHAB configured timezone and also different to the persistence service. Could you elaborate on why this does not matter or is the right timezone?

Reply:
This has been in the past the same, only at a different class: https://github.com/openhab/openhab-addons/pull/17578/files/2f1c7241eb63de738a8eb4c568f1d12d0b6ab627#diff-05fde9430a9ef73b9401cdb1375bc43bd99b501679e037d24e67eb5717208bf4L259

In the future it might be good to deprecate `getTimestamp`